content/secure-remote-access/README.md

```diff
@@ -0,0 +1,56 @@
+# Secure Remote Access
+
+## Overview
+Instead of using qidi link to access your printer, this guide servces to help you 
+setup a VPN mesh network that allows you to access your printer using various klipper
+apps for mobile devices using your trusted network as opposed to using Qidi's. 
+
+## Prerequisites
+* Comfortable with SSH access to your printer and executing commands
+* Have an free tailscale account - https://tailscale.com/signup/
+
+## 1. Setup Tailscale VPN Account
+Follow the onboarding guidelines from tailscale to create and register your first device, preferabbly your phone since you are replacing qidi link! Instructions are [here](https://tailscale.com/kb/1017/install). Once you see your phone listed in the devices, move on to the next step!
+
+## 2. Disable qidi link
```

Comment:
Enabling lan-only mode disables qidi link systemd service and also stops xindi from trying to generate authentication codes every once in a while. This would be much preferred.

Reply:
it is also quite optional considering tailscale *really* doesn't conflict with qidi link, even though it replaces it.

Reply:
I think this is a good call out as well. If LAN only mode disables the QIDILink-client.service, that'll free up resources. Will update documentation to reflect this.

Reply:
@vega-d Updated this section to instead only call out enabling lan only mode for simplicity and also call it out as an optional step

https://github.com/qidi-community/Plus4-Wiki/pull/31/commits/c20e07d9808c666a76b7369fe69fa24afd48d01e
https://github.com/qidi-community/Plus4-Wiki/pull/31/commits/70975ed3edb8abc6ecea53feb8e521faa1436c01